autoload/cmake4vim.vim

```diff
@@ -176,7 +193,13 @@ endfunction
 
 " Functions allows to switch between build types
 function! cmake4vim#SelectBuildType(buildType) abort
-    let g:cmake_build_type = a:buildType
+    " so it's called only once
+    if ( !exists( 's:initialized_default_build_types' ) )
+        call cmake4vim#populateDefaultCMakeVariants()
```

Comment:
I think we can call this method every time, because user can change cmake build types configuration in the runtime.